test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
It should be quite easy for me to implement both ways to do it.
The only question is: should I add an option on default_include (`root_only`, or `always`), or should I make another class attribute to support both of them 😃 ?